cmd/eksctl-anywhere/cmd/generatepackage.go

```diff
@@ -27,6 +29,9 @@ func init() {
 	generatePackageCommand.Flags().StringVar(&gpOptions.registry, "registry", "", "Used to specify an alternative registry for package generation")
 	generatePackageCommand.Flags().StringVar(&gpOptions.kubeConfig, "kubeconfig", "",
 		"Path to an optional kubeconfig file to use.")
+	if err := generatePackageCommand.MarkFlagRequired("cluster"); err != nil {
+		log.Fatalf("marking cluster flag as required: %s", err)
```

Comment:
Now I am getting concerned with requiring a cluster flag in case we are trying to generate before a cluster is created

Reply:
before a cluster created is registry discovery 🤷

Reply:
Anyway, I'm not dealing with this today. Tests fail for me on main, so I didn't notice these borked tests.